cmd/apps.go

```diff
@@ -16,9 +16,26 @@ import (
 	"github.com/deis/workflow-cli/settings"
 )
 
+// AppCmd is interface definition for running app commands
+type AppCmd interface {
```

Comment:
maybe once you roll this out to the rest of `cmd` you might want to rename this to `Commander`. I imagine we'd want a single interface that defines every command in the `cmd` package for mocking purposes so we don't have to create specific structs for every subcommand.